src/devices/bus/coco/coco_multi.cpp

```diff
@@ -126,7 +126,7 @@ namespace
 		// methods
 		void set_select(uint8_t new_select);
 		DECLARE_WRITE8_MEMBER(ff7f_write);
-		void update_line(int slot_number, cococart_slot_device::line line);
+		void update_line(int slot_number, line line);
```

Comment:
Oh my god, local shadows type name.  No, just no.  Call it `value` or something else, this is a recipe for confusion.